Demo/Podfile.lock

```diff
@@ -13,11 +13,11 @@ DEPENDENCIES:
 
 EXTERNAL SOURCES:
   NSObject+Rx:
-    :path: "../"
+    :path: ../
```

Comment:
I think this needs to be `../NSObject_Rx` to make CI happy.

Reply:
No, this points to the podspec so it should be fine